qiita_db/processing_job.py

```diff
@@ -488,9 +490,34 @@ def get_resource_allocation_info(self):
                     AssertionError(
                         "Could not match %s to a resource allocation!" % name)
 
-            # [0] sending one element as execute_fetchflatten returns
-            # an array
-            return result[0]
+            allocation = result[0]
+
+            if ('{samples}' in allocation or '{columns}' in allocation or
+                    '{input_size}' in allocation):
+                samples, columns, input_size = self.shape
+                parts = []
+                for part in allocation.split():
```

Comment:
What is the split being done on? Are we expecting multiple lines per allocation?

Reply:
by default is done by space ...

Reply:
Forgot about that.